docs/book/en/src/sdk/go/0.9.1/ref.md

```diff
@@ -23,13 +23,16 @@ The followings are the guides to working with the WasmEdge-Go SDK at WasmEdge ve
   * [Results](#results)
   * [Contexts And Their Life Cycles](#contexts-and-their-life-cycles)
   * [WASM data structures](#wasm-data-structures)
+  * [Async](#async)
```

Comment:
Not pretty sure why do we need to update the 0.9.1 document?

Reply:
The docs of v0.9.0 lost the implemented APIs.